lib/eco/treelexer/lexer.py

```diff
@@ -132,6 +120,12 @@ def match(self, pattern, text):
             return self.match_question(pattern, text)
         raise NotImplementedError(pattern)
 
+    def match(self, pattern, text):
```

Comment:
I'm not a fan of having `match` and `_match` -- that's just too close a typo to make. Can we rename one method to something a bit more obviously distinct?

Reply:
Sure I couldn't think of anything tho. `_match` used to be `match` before I realised I needed another method to be called `match`. I welcome any suggestions.

Reply:
Nearly anything'll be better ;) Even `_inner_match` or something like that.